app/src/main/java/de/stephanlindauer/criticalmaps/fragments/ChatFragment.java

```diff
@@ -104,12 +103,12 @@ public void onActivityCreated(final Bundle savedState) {
     public void onViewStateRestored(@Nullable Bundle savedInstanceState) {
         super.onViewStateRestored(savedInstanceState);
 
-        sendButton.setEnabled(editMessageTextField.getText().length() > 0);
+        updateSendButtonEnabledState();
```

Comment:
This line should not be changed. Otherwise - if the fragment is recreated with the previous state, e.g. after device rotation - the button will be animated again.

Reply:
Of course the .trim() should also be added there.

Reply:
@cbalster check again please. I also added final modifiers for some fields

Reply:
Nice, thanks!